src/Xamarin.Android.Build.Tasks/Tasks/LinkApplicationSharedLibraries.cs

```diff
@@ -186,6 +187,14 @@ IEnumerable<Config> GetLinkerConfigs ()
 					}
 				}
 
+				uint maxPageSize = ZipAlignmentPages switch {
+					4  => 4096,
+					16 => 16384,
+					_  => throw new InvalidOperationException ($"Internal error: unsupported zip page alignment value {ZipAlignmentPages}")
```

Comment:
I guess we'd use `LogError()` here too, if we changed the other one.

Reply:
I probably should just move this switch to a method inside `MonoAndroidHelper` class.